reporter-xray-udp/README.md

```diff
@@ -1,3 +1,69 @@
-# reporter-xray-udp
+# AWS X-Ray Reporter for Zipkin
 
-This sends json to XRay's daemon via UDP
\ No newline at end of file
+Used in an application that is being traced, this component converts Zipkin 
+trace data (spans) into [AWS X-Ray](https://aws.amazon.com/xray/)'s
+proprietary format and sends them to the 
+[X-Ray daemon](https://docs.aws.amazon.com/xray/latest/devguide/xray-daemon.html) 
+using UDP.
+
+Note that, unlike `AsyncReporter`, this reporter attempts to encode and send
+the span immediately on the calling thread.
+As UDP is used, there is no latency in waiting for the daemon to accept and 
+respond to the data.
```

Comment:
Might be good to note, that like all other types of `Reporter` it can easily be wrapped in an `AsyncReporter`

Reply:
I like the idea, but I can't see any way to wrap a Reporter with an AsyncReporter. AsyncReporter.builder() seems only to accept a Sender. If you're sure this is possible, could you give me an example of how to do it?

Reply:
Ah sorry. You are correct. I rushed through reviewing